src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE books (
+id BIGINT AUTO_INCREMENT PRIMARY KEY,
+title VARCHAR(255) NOT NULL,
+price DECIMAL(10, 2) NOT NULL

```

Comment:
minor, but consider adding some tabulation to distinguish SQL stats themselves and fields declaration